google/ggrc-core
# QUICK-FIX Fix deployment via kokoro

# Issue description

*Explain the issue that the changes in the pull request solve.*
Currently deployment via kokoro always deploys to appengine version "1". Even if the version is set to something else or "AUTO", it always deploys to version "1". There are two reasons for it.
1. Variable VERSION is missing in kokoro release config (kokoro/ggrc_release.sh)
2. After adding VERSION variable in ggrc_release.sh, it agains fails complaining about FETCH_HEAD is not set (bin/do_deploy).

# Steps to test the changes

In order to test this change follow following steps.

1. Merge this PR on top of last stable release. 
2. Go to kokoro UI and under release_job use the test instance (sandbox for example). If you want to use your forked repo for testing this then let me know.
Note : The settings for each instance is available in "ggrc_test_bucket" bucket under ggrc-qa instance. Make sure you add VERSION="AUTO" for auto incrementing version or your custom version.
3. Click Build with parameters from left panel, supply the commit hash and build.

# Solution description

To solve above issue, I have added missing VERSION variable in kokoro/ggrc_release.sh and added a condition to distinguish the source of deployment (kokoro or local machine). If deployment is done via kokoro then it uses HEAD otherwise it uses the usual FETCH_HEAD in bin/do_deploy script.

# Sanity checklist




# PR Review checklist
